content/it/docs/concepts/containers/container-environment.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Container Environment
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Questa pagina descrive le risorse disponibili nei Containers eseguiti in Kubernetes.
```

Comment:
s/Containers/Container/ (in Italian the plural form of English words is made with plural article + singular English noun)